### DISCUSSION TITLE
About github license

### DISCUSSION THREAD

**Discussion by popstar7** [Finalgoal231/.github#10] [Show and tell] [answered]
Please explain licence file.
Licence file need to readme.md of github?

**Comment by noahstoner** [comment, marked as answer]
License means what allowed or not allowed.
So with this, we can use in some case or can't use in some case

**Reply by popstar7** [reply]
Oh, I got it

**Comment by amirallami-code** [comment]
> Please explain licence file. Licence file need to readme.md of github?

Hi @popstar7, if you want to know more about License and licensing you can read this guide, I’m trying to cover important information about license [Link](https://github.com/amirallami-code/license-guide)
